### PR TITLE
Disallow network requests in tests

### PR DESCRIPTION
**Pull request by keithjgrant**
##### SUMMARY
- enforces no network requests are made during unit tests, by forcing test failure if it occurs
- cleans up tests that were initiating network requests

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
 - UI

### CODE REVIEW

**Review by nixocio — APPROVED** [review]

File: awx/ui_next/src/screens/User/UserTokens/UserTokens.test.jsx [hunk 1]
```diff
@@ -5,14 +5,11 @@ import { mountWithContexts } from '../../../../testUtils/enzymeHelpers';
 
 import UserTokens from './UserTokens';
 
+jest.mock('../../../api');
+
```
Comment (tiagodread) [inline awx/ui_next/src/screens/User/UserTokens/UserTokens.test.jsx]: Just a question, is this mocked but not used?

**Review by tiagodread — APPROVED** [review]

**Comment by tiagodread** [comment]
@keithjgrant could you rebase this one and confirm if a retest will be necessary?

**Comment by keithjgrant** [comment]
ah shoot, I didn't look close enough at that rebase... let me clean that up a bit

**Comment by tiagodread** [comment]
@keithjgrant There is one failing test
```
FAIL src/screens/Setting/SettingList.test.js
  ● Test suite failed to run

    Cannot find module '../../util/useBrandName' from 'src/screens/Setting/SettingList.test.js'

      4 |
      5 | jest.mock('../../api');
    > 6 | jest.mock('../../util/useBrandName', () => ({
        |      ^
      7 |   __esModule: true,
      8 |   default: () => ({
      9 |     current: 'AWX',

      at Resolver.resolveModule (node_modules/jest-runtime/node_modules/jest-resolve/build/index.js:306:11)
      at Object.<anonymous> (src/screens/Setting/SettingList.test.js:6:6)
```

**Pull request merged**